### PR TITLE
Corrected spellings in 1st para and added a line in the 2nd para

### PR DESCRIPTION
**Pull request by dangerouspython**
Hello, just corrected your spellings in the 1st para and added a line that you had missed in the 2nd para

### CODE REVIEW

File: road_not_taken.txt [hunk 1]
```diff
@@ -1,11 +1,12 @@
-Two road diverged in a yellow wood,
-And sorry I could travel both
-And be a traveler, long I stood
+Two roads diverged in a yellow wood,
+And sorry I could not travel both
+And be one traveler, long I stood
 And looked down as far as I could
 To where it bent out in the undergrowth;
 
 Then took the other as just as fair,
```
Comment (malignantpython) [inline road_not_taken.txt]: Hi! could you correct the spellings/punctuation here?

File: road_not_taken.txt [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 And having perhaps the better claim,
 Because it was grassy and wanted wear;
+Though as for that the passing there
 Had worn them really about the same
```
Comment (malignantpython) [inline road_not_taken.txt]: And here too?

**Comment by malignantpython** [comment]
Only then I can accept the pull request :smile:

**Comment by dangerouspython** [comment]
Yeah sure!

**Comment by dangerouspython** [comment]
Is there anything more that I can do?
- [X] Corrected punctuations
- [ ] Corrected spelllings
- [ ] Add remaining paragraphs
- [ ] Extra

**Comment by malignantpython** [comment]
Thanks! No nothing for now.. :smiley:

**Pull request merged**